classes/components/forms/publication/ContributorForm.php

```diff
@@ -94,10 +95,47 @@ public function __construct(string $action, array $locales, Submission $submissi
             ]))
             ->addField(new FieldText('url', [
                 'label' => __('user.url'),
-            ]))
-            ->addField(new FieldText('orcid', [
-                'label' => __('user.orcid'),
             ]));
+
+        if (OrcidManager::isEnabled()) {
+            $this->addField(new FieldText('orcid', [
+                'label' => __('user.orcid'),
+                'optIntoEdit' => true,
+                'optIntoEditLabel' => __('common.override'),
```

Comment:
I think in general, no, we do not want to allow manual editing. Could you share a reference to the code where it's validated at publication (just want to make sure I'm considering the right thing).

I think the ideal solution would be to change this from a form field to a "static field." I've talked with Devika about using this pattern for the context-level client ID/secret if it's set at the context level. It will still display information inline in what appears like a form field to the user, but is just a a text field underneath, and since that's how we're saving the ORCIDs, that's probably the best option.

For legacy, unauthorized ORCIDs, these will need to be indicated as such somehow (I'll need to review previous version of the plugin to see how that's handled), but I don't think letting users directly edit them in any case will be needed, since, ideally, if something is wrong, they can delete the legacy, unauthenticated ORCID, and then go through the authentication process as usual.

Reply:
I classes/publication/Repository.php, there is this code:
```
elseif ($authorOrcid && !$author->getData('orcidAccessToken')) {
$errors['hasUnauthenticatedOrcid'] = __('orcidProfile.verify.hasUnauthenticatedOrcid');
}
```
I thought this is the check... :-)